mms/export_model.py

```diff
@@ -44,8 +44,8 @@
 SIG_REQ_ENTRY = ['inputs', 'input_type', 'outputs', 'output_types']
 VALID_MIME_TYPE = ['image/jpeg', 'application/json']
 SIGNATURE_FILE = 'signature.json'
-MODEL_ARCHIVE_VERSION = 0.1
-MODEL_SERVER_VERSION = 0.1
+MODEL_ARCHIVE_VERSION = 0.2
```

Comment:
Why do we need an archive version that is different from the version of the export utility creating it?

Reply:
This isn't currently used. I just modified it to track significant the changes to the tool.